core/domain/question_domain.py

```diff
@@ -296,3 +319,35 @@ def to_dict(self):
             'question_id': self.question_id,
             'skill_id': self.skill_id,
         }
+
+
+class QuestionRights(object):
+    """Domain object for question rights."""
+
+    def __init__(self, question_id, manager_ids):
+        self.id = question_id
+        self.manager_ids = manager_ids
+
+    def to_dict(self):
+        """Returns a dict suitable for use by the frontend.
+
+        Returns:
+            dict. A dict version of QuestionRights suitable for use by the
+                frontend.
+        """
+        return {
+            'question_id': self.id,
+            'manager_names': user_services.get_human_readable_user_ids(
```

Comment:
Why do you need manager names? Again, verify whether you actually need them.

Reply:
I was thinking that the admins and topic manager and the user who created the question would be a part of manager_names